packages/pagination/src/styled/OffsetPagination/StyledPage.ts

```diff
@@ -17,7 +17,7 @@ const sizeStyles = (props: ThemeProps<DefaultTheme>) => {
 
   return css`
     min-width: ${height};
-    max-width: ${math(`${height} * 2`)}; /* [1] */
+    max-width: ${math(`${height} * 2`)};
```

Comment:
Should we get rid of the expensive `math` function by making `height` a number calculation and appending `'px'` where necessary?